x-pack/plugins/cases/public/components/cases_context/index.tsx

```diff
@@ -114,10 +125,35 @@ export const CasesProvider: React.FC<{ value: CasesContextProps }> = ({
     }
   }, [appTitle, appId]);
 
+  const applyFilesContext = useCallback(
+    (contextChildren: ReactNode) => {
+      if (owner.length === 0) {
+        return contextChildren;
+      }
+
+      if (owner[0] in CASES_FILE_KINDS) {
+        return (
+          <FilesContext client={getFilesClient(CASES_FILE_KINDS[owner[0] as Owner].id)}>
+            {contextChildren}
+          </FilesContext>
+        );
+      } else {
+        throw new Error(
+          'Invalid owner provided to cases context. See https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/README.md#casescontext-setup'
```

Comment:
Do we need to translate this? I assume this would be a development error if it occurred right?

Reply:
My intent was to inform devs if they misconfigure the case context. I don't think users will see it.

[There is something similar happening here.](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/components/cases_context/use_cases_context.ts#L16)